### PR TITLE
Adding manifest for 3CX.3CXCallFlowDesigner

### PR DESCRIPTION
**Pull request by gustavosaquino**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16995)

### CODE REVIEW

**Comment by gustavosaquino** [comment]
During testing, this installer did not support silent or very silent mode without progress.

File: manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - interactive
+  - silent
+  - silentWithProgress
+Installers:
+  - Architecture: x64
+    InstallerType: exe
+    InstallerUrl: https://downloads-global.3cx.com/downloads/3CXCallFlowDesigner.exe
+    InstallerSha256: D852E35FAC3ACDFAEE2E127D8AE77724C1FCD952982296D4CDE933C82546E03D
+    InstallerSwitches:
+#      Silent:
```
Comment (OfficialEsco) [inline manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.installer.yaml:17]:
```suggestion
      Silent: /qb
```
or /qn if that works

File: manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: 3CX.3CXCallFlowDesigner
+PackageVersion: 16.6.53
+PackageLocale: en-US
+Publisher: 3CX
```
Comment (OfficialEsco) [inline manifests/3/3CX/3CXCallFlowDesigner/16.6.53/3CX.3CXCallFlowDesigner.locale.en-US.yaml:5]:
```suggestion
Publisher: 3CX Ltd
```
Needs to match what control panel sees

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please do those quick changes :)

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**